_posts/luznette/2020-09-15-Janette-turtle-hack.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+author: luznette
+title: Janette's Turtle Hack
+---
+For this assignment, I wanted to work on arguments/statements that I did not fully understand. The program that I invisioned would give the user a few prompts to choose a background color, the color of the pen, and a shape to draw. After it drew the shape, I wanted it to ask the user if they wanted to draw again. If they chose "no", then the program would end. If they chose "yes", then the screen would clear and the prompts would come back up again until the user chose "no". For some reason, I could not figure out how to clear the screen before it drew again. At least it loops!
+
+In the procress, I quickly realized that I am in the population that needs to understand how things work before I can try them out. I would often pause to search youtube and [w3schools](http://w3schools.com/) for different explanations. It was not until I had spent a few hours on the assignment that I let that feeling go. I believe my biggest challenge is not understanding lines of code if it is not written in the way it is executed. For example, when writing my statements, my initial thought was to write it as:
+```
+print ("Hello, my name is Tina. Today we are drawing shapes.")
+thisShape = instructions()
+draw(thisShape)
+ask = input("Would you like to draw again? (Yes or No):")
+while ask=="yes" or ask=="Yes":
+  thisShape = instructions()
+  draw(thisShape)
+  ask = input("Would you like to draw again? (Yes or No):")
+else:
+  print ("Thank you!")
+```
```

Comment:
I like the way you explain your initial thought here. They do help me understand your code and what you were trying to achieve!